apps/financial-aid/web-osk/src/components/Estimation/Estimation.tsx

```diff
@@ -94,30 +55,13 @@ const Estimation = ({
 
       {aboutText}
 
-      {municipality && (
-        <>
-          {calculations.map((item, index) => {
-            return (
-              <span key={`calculations-` + index}>
-                <Divider />
-                <Box
-                  display="flex"
-                  justifyContent="spaceBetween"
-                  alignItems="center"
-                  padding={2}
-                  background={
-                    index === calculations.length - 1 ? 'blue100' : 'white'
-                  }
-                >
-                  <Text variant="small">{item.title}</Text>
-                  <Text>{item.calculation}</Text>
-                </Box>
-              </span>
-            )
-          })}
-
-          <Divider />
-        </>
+      {municipality && aidAmount && (
```

Comment:
Should breakdown rather take these variables and retur null if they are false like we do in other places?